src/python/pants/backend/python/util_rules/BUILD

```diff
@@ -13,7 +13,7 @@ python_tests(
     name="tests",
     overrides={
         "local_dists_test.py": {"timeout": 120},
-        "pex_from_targets_test.py": {"timeout": 200},
+        "pex_from_targets_test.py": {"timeout": 200, "dependencies": ["3rdparty/python#pex"]},
```

Comment:
Not as part of this PR, but after seeing the errors, thoughts on the LoE to remove the test dependency?  *Eventually* I think we will want to consider using the scie Pex distributions instead of the PyPI wheels.

Reply:
I think the main thing that is required to get these tests working without the dependency is finding a way to get pex inside the sandbox without explicitly requiring it. We may be able to potentially find another way to do that, but that's the crux issue for why these tests fail.